staging/src/k8s.io/api/go.mod

```diff
@@ -5,12 +5,15 @@ module k8s.io/api
 go 1.12
 
 require (
-	github.com/gogo/protobuf v0.0.0-20171007142547-342cbe0a0415
-	github.com/stretchr/testify v1.2.2
+	github.com/gogo/protobuf v1.1.1
+	github.com/stretchr/testify v1.3.0
 	k8s.io/apimachinery v0.0.0
 )
 
 replace (
+	github.com/google/uuid => github.com/google/uuid v1.0.0
+	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.6.0
```

Comment:
hmm... seeing a lot of new replace statements in staging go.mod files, which is not good